src/components/Book.js

```diff
@@ -3,40 +3,66 @@ import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import Button from './Button';
 import { removeBook, removeAsyncBook } from '../redux/books/booksSlice';
+import './styles/book.css';
 
-function Book({ id, title, author }) {
+function Book({
+  id, title, author, category,
+}) {
   const dispatch = useDispatch();
   const handelRemoveBook = () => {
     dispatch(removeAsyncBook(id));
     dispatch(removeBook(id));
   };
   return (
-    <>
-      <div>
-        <ul>
+    <div className="Lesson-Panel">
+      <p className="school-of">
+        {category}
+        {' '}
+      </p>
+      <h4 className="title">{title}</h4>
+      <p className="suzanne-collins">{author}</p>
+      <div className="flex-spac-btwn">
+        <div className="buttons-contanier flex">
+          <Button className="comments-btn" value="Comment" type="button" />
+          <hr />
```

Comment:
- Kindly fix the grey line between options to apply the same style as on the Zeplin design
- Kindly change the font style to match zeplin's design

Your design

![image](https://github.com/naji-97/sec-book-store/assets/93630700/6b213866-8278-4d0a-b889-b2aa9c6ed1f9)

Zeplin design

![image](https://github.com/naji-97/sec-book-store/assets/93630700/8043d385-2d77-41cd-848b-e7a46c5b8c33)